src/circuit/reservation-store.ts

```diff
@@ -38,8 +46,10 @@ export class ReservationStore implements IReservationStore, Startable {
     this.init = {
       maxReservations: options?.maxReservations ?? 15,
       reservationClearInterval: options?.reservationClearInterval ?? 300 * 1000,
-      applyDefaultLimit: options?.applyDefaultLimit === false,
```

Comment:
This is fun, it would have turned every js-libp2p node with the relay server enabled into an unlimited relay 😱